<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="77X35R1-LT" name="com.careyconnect.CareyConnectTest" tests="1" failures="0" timestamp="8 Apr 2020 20:36:56 GMT" time="0.281" errors="1">
  <testcase name="testLogin" time="0.281" classname="com.careyconnect.CareyConnectTest">
    <error type="org.testng.TestNGException" message="
The data provider is trying to pass 6 parameters but the method com.careyconnect.CareyConnectTest#testLogin takes 1">
      <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 6 parameters but the method com.careyconnect.CareyConnectTest#testLogin takes 1
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1232)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1125)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- testLogin -->
</testsuite> <!-- com.careyconnect.CareyConnectTest -->
